src/WebJobs.Script.WebHost/Management/TriggersPayload.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Management
+{
+    public sealed class TriggersPayload : TriggersOperationResult
+    {
+        /// <summary>
+        /// Gets or sets the triggers payload.
+        /// </summary>
+        public string Content { get; set; }
```

Comment:
I am not a fan of using `string Content` here. I much prefer returning a serializable object and letting the MVC formatters serialize it. I also get the current implementation does manual serialization and `JObject` interactions.

Can you evaluate the work involved with changing this?

Reply:
Agree usually, but in this case this was done because [GetSyncTriggersPayload](https://github.com/Azure/azure-functions-host/blob/7e3e7b78fa90ec18eaefd995a9e7797d326d2d00/src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs#L309) exposes a string contract because internally it's doing size checks/truncation. I'd rather not go refactor all that right now. The work would be to change the contract of the existing SyncTriggersPayload, and make callers responsible for json formatting etc. Currently that logic is centralized in this method.

Reply:
> Another point here is that we need to maintain parity between this new GET API, and the existing SyncTriggers POST API, so we want to use the same serialization in both cases.

It makes no sense for a GET api to double serialize JSON. The POST API makes sense because we are writing that to storage, it is not part of the API response. Including the `string Content` in the API response though is a whole different story and we **should not** do that. You are going to have this response:

``` json
{
    "content": "<json-serialized-string>"
}
```

That to me is not ideal. Instead we could have:

``` json
{
    "content" : {
       // json serialized content of sync triggers.
    }
}
```

Reply:
Mainly, we are using this API to perform sync triggers during deployment so we want the payload to be exactly what sync triggers is using.

Reply:
Added remark:
```
/// <summary>
/// Gets or sets the triggers payload.
/// </summary>
/// <remarks>
/// Use of an already formatted JSON string here as opposed to a JObject is by design.
/// We want to ensure that the result we return here is the exact same JSON payload as is generated during a sync triggers operation (down to whitespacing).
/// </remarks>
public string Content { get; set; }